10/alpine3.10/Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN addgroup -g 1000 node \
       && case "${alpineArch##*-}" in \
         x86_64) \
           ARCH='x64' \
-          CHECKSUM="f893a03c5b51e0c540e32cd52773221a2f9b6d575e7fe79ffe9e878483c703ff" \
+          CHECKSUM="" \
```

Comment:
These mean the Unofficial builds for them haven't finished yet

Reply:
they're out now